Facebook.Android/source/NuGet.Config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
```

Comment:
This is OK. Only local feed but for combination with nuget.org `clear` would be needed:

https://github.com/xamarin/AndroidX/blob/master/samples/NuGet.config#L4-L6